staging/src/k8s.io/kubectl/pkg/cmd/create/create_configmap.go

```diff
@@ -17,12 +17,27 @@ limitations under the License.
 package create
 
 import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+	"unicode/utf8"
+
 	"github.com/spf13/cobra"
 
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/validation"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/kustomize/k8sdeps/transformer/hash"
```

Comment:
This is wrong, should be `"k8s.io/kubectl/pkg/util/hash"`

Reply:
ok!